test/conversations/Conversation.test.ts

```diff
@@ -309,20 +311,19 @@ describe('conversation', () => {
       const stream = await bobConvo.streamMessages()
       await sleep(100)
       // mallory takes over alice's client
-      const malloryWallet = newWallet()
-      const mallory = await PrivateKeyBundleV1.generate(malloryWallet)
-      const aliceKeys = alice.legacyKeys
-      alice.legacyKeys = mallory
+      const mallory = await newLocalHostClient()
+      const aliceKeystore = alice.keystore
+      alice.keystore = mallory.keystore
       await aliceConvo.send('Hello from Mallory')
       // alice restores control
-      alice.legacyKeys = aliceKeys
+      alice.keystore = aliceKeystore
       await aliceConvo.send('Hello from Alice')
       const result = await stream.next()
       const msg = result.value as DecodedMessage
       expect(msg.senderAddress).toBe(alice.address)
       expect(msg.content).toBe('Hello from Alice')
       await stream.return()
-      expect(consoleWarn).toBeCalledTimes(1)
+      expect(consoleWarn).toBeCalledTimes(2)
```

Comment:
dumb question: where does the second warn come from?

Reply:
Was a change to the sequence of where we log decryption errors. But looking back at it, we probably don't need to log twice in this case. Moved the console.log a few lines down and it's back to 1.